outputs.bmeg_manifest.dvc

```diff
@@ -308,140 +308,136 @@ deps:
   path: outputs/tcga/UVM.Expression.Vertex.json.gz
 - md5: 0abcb1595fab9c96d58d6baccd9ed65a
   path: outputs/tcga/UVM.ExpressionOf.Edge.json.gz
-- md5: 351b7225441a45a88dc2b89b9be94c2c
-  path: outputs/tcga/TCGA-ACC.CopyNumberAlteration.Vertex.json
-- md5: d99490356b48d72e62556fd7c3471cb7
+- md5: eb869fafc48d8476f91175ff5381d858
   path: outputs/tcga/TCGA-ACC.CopyNumberAlteration.Vertex.json.gz
-- md5: 1dc3ddcebd7185200844e8fc980a40c8
-  path: outputs/tcga/TCGA-ACC.CopyNumberAlterationOf.Edge.json
-- md5: 5bf0299eee8488abb12188abe419c5f8
+- md5: ce9cfd1818014fb1425856d90bb375a5
   path: outputs/tcga/TCGA-ACC.CopyNumberAlterationOf.Edge.json.gz
-- md5: b27e006e080d38763b179521d0b1ede9
+- md5: 41d0ae4615f2f0b537962846cfef4465
```

Comment:
Any idea why all of these md5 sums changed?

Reply:
I'm not sure.   I pulled from master, pulled from (dvc minio) and did a `dvc repro outputs.bmeg_manifest.dvc` and it indicated gistic2cna needed to be re-run.